aws/rds-global-cluster/variables.tf

```diff
@@ -0,0 +1,111 @@
+variable "source_region" {
+  type        = string
+  description = "The source region for an encrypted replica DB cluster."
+}
+
+variable "target_region" {
+  type        = string
+  description = "TThe target region for an encrypted replica DB cluster."
+}
+
+
+variable "global_cluster_identifier" {
+  type        = string
+  description = "(Required, Forces new resources) Global cluster identifier."
+}
+
+
+variable "engine" {
+  type        = string
+  description = "Name of the database engine to be used for this DB cluster. Terraform will only perform drift detection if a configuration value is provided. Valid values: aurora, aurora-mysql, aurora-postgresql."
+}
+
+variable "engine_version" {
+  type        = string
+  description = "Engine version of the Aurora global database."
+}
+
+variable "database_name" {
+  type        = string
+  description = "(Optional, Forces new resources) Name for an automatically created database on cluster creation."
```

Comment:
I think it's not optional as it's defined here with non `default` value right? It will be always required by terraform. ideally if we want to make it optional just add `default=""`